docs/dev_guide/stability.rst

```diff
@@ -90,7 +90,7 @@ The current planned and existing sub-packages are:
     {% for module, prop in sunpy_modules.items() %}
         <tr>
             <td>
-                <a href="code_ref/{{ module }}.html">sunpy.{{ module }}</a>
+                <a href="{{ module }}.html">sunpy.{{ module }}</a>
```

Comment:
it seemed to work for me locally if I do `./code_ref/{{ module }}.html`

Reply:
Oh, Ok. I will try that

Reply:
This is not working because if we do `./code_ref/{{ module }}.html` we get the following link
`file:///home/naman/sunpy-git/docs/_build/html/dev_guide/code_ref/cm.html` but we need to go to
`file:///home/naman/sunpy-git/docs/_build/html/code_ref/cm.html`

Reply:
Ah it did not render correctly. It was meant to be two dots.

Reply:
Yup I was about to say that because that worked for me